crates/core/src/repository.rs

```diff
@@ -356,11 +341,11 @@ impl<P> Repository<P, ()> {
         let mut be = backends.repository();
         let be_hot = backends.repo_hot();
 
-        if !opts.warm_up_command.is_empty() {
-            if opts.warm_up_command.iter().all(|c| !c.contains("%id")) {
+        if let Some(warm_up) = &opts.warm_up_command {
+            if !warm_up.args().iter().all(|c| !c.contains("%id")) {
```

Comment:
Why is this negated now? 🤔

Reply:
Good catch - that was a typo.
I fixed it and also added an integration test which would have caught this.